release-calendar/src/types.ts

```diff
@@ -51,4 +51,5 @@ export class Promotion {
 export interface ApiService {
   getRelease: (name: string) => Promise<Release>;
   getReleases: () => Promise<Release[]>;
+  getCurrentReleases: () => Promise<Promotion[]>;
```

Comment:
The return type is different than the method in api-service.ts?

Reply:
no, that was just my mistake. I kept switching between returning promotions or releases :( I will try to be more thorough in my personal reviews.